guidelines/commit_and_pr_text.md

```diff
@@ -0,0 +1,55 @@
+# Motivation
+Our changelogs (reviewed by users/customers) are autogenerated from PR titles and git commit text.  The better this information is, the more we help our customers.  This information also helps reviewers understand changes, patch managers understand risk/importance, and when hunting for regressions, helps us better understand what is/isn’t likely to be a cause.
+
+# PR Metadata
+
+__PR title:__ Bug NNNN: A short sentence with capitalization and no period or the Jira tag for the issue “Feature TEAM-XXX:”
```

Comment:
Problem for monitoring team is we have a lot of dependancies that are bumped with one single PR, when updating jsonnet deps, so how would we describe this in short? We now say something vague like "Bump dependencies". What would you suggest for us to write here instead?

Reply:
I think that's a totally reasonable summary sentence for a dep bump.

however if there's anything special worthy of note about the bump, it can be put in the longer description (e.g. did you have a pin a dep version or add a new one?  did other code have to refactored in reaction to the bump, in which case those changes should probably be a separate commit in the same PR?)

Reply:
Maybe something like https://github.com/openshift/cluster-kube-controller-manager-operator/pull/406#issue-409539505, maybe not necessarily as detailed.

Reply:
Nice, that makes sense. Any objections @openshift/openshift-team-monitoring maybe we can just find a way to somehow copy over the commits from the upstream deps, e.g. kube-mixin commits?